kubewarden_test.go

```diff
@@ -0,0 +1,339 @@
+package sdk
+
+import (
+	"testing"
+
+	appsv1 "github.com/kubewarden/k8s-objects/api/apps/v1"
+	batchv1 "github.com/kubewarden/k8s-objects/api/batch/v1"
+	corev1 "github.com/kubewarden/k8s-objects/api/core/v1"
+	"github.com/kubewarden/policy-sdk-go/protocol"
+	"github.com/mailru/easyjson"
+	"github.com/mailru/easyjson/jwriter"
+)
+
+func CreateValidationRequest(object easyjson.Marshaler, kind string) (protocol.ValidationRequest, error) {
```

Comment:
nitpicking, do we need that to be exported? I would just call it `createValidationRequest`